web/src/App.test.js

```diff
@@ -1,6 +1,6 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
-
+// CR * cleaning up boilerplate
```

Comment:
Do you need this test file? I don't think so. Eventually, you want to make sure your code is lean and mean, and that may entail removing unnecessary files added from your boilerplate (in this case, `create-react-app`).